<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analyze Garmin GPX files</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="dirTreeView"/>
      <widget class="QTableView" name="trkListView">
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QTableView" name="wptListView">
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QTableView" name="rteListView">
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QTableView" name="fitListView">
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="loadGpxButton">
        <property name="text">
         <string>&amp;Load Garmin Devices</string>
        </property>
        <property name="shortcut">
         <string>Alt+L</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openGpxButton">
        <property name="text">
         <string>&amp;Open GPX file</string>
        </property>
        <property name="shortcut">
         <string>Alt+O</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteGpxButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string extracomment="Delete selected GPX file"/>
        </property>
        <property name="text">
         <string>&amp;Delete GPX File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteFitButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete &amp;FIT File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>E&amp;xit</string>
        </property>
        <property name="shortcut">
         <string>Alt+X</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
